src/main/java/com/alvarium/serializers/AnnotationConverter.java

```diff
@@ -14,24 +13,22 @@
  *******************************************************************************/
 package com.alvarium.serializers;
 
-import java.lang.reflect.Type;
-
 import com.alvarium.contracts.Annotation;
-import com.google.gson.JsonDeserializationContext;
-import com.google.gson.JsonDeserializer;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParser;
-import com.google.gson.JsonSerializationContext;
-import com.google.gson.JsonSerializer;
+import com.google.gson.*;
+
+import java.lang.reflect.Type;
 
+public class AnnotationConverter implements JsonSerializer<Annotation> {
 
-public class AnnotationConverter implements JsonSerializer<Annotation>, JsonDeserializer<Annotation> {
+    @Override
+    public JsonElement serialize(Annotation src, Type typeOfSrc, JsonSerializationContext context) {
+        var json = new JsonObject();
 
-  public JsonElement serialize(Annotation src, Type typeOfSrc, JsonSerializationContext context) {
-    return JsonParser.parseString(src.toJson()); 
-  }
+        json.add("id", new JsonPrimitive(src.getId()));
+        json.add("type", new JsonPrimitive(src.getKind().name()));
+        json.add("tag", new JsonPrimitive(src.getTag()));
+        json.add("isSatisfied", new JsonPrimitive(src.getIsSatisfied()));
```

Comment:
The annotations being produced look like this:
`{"id":"01HZPVF01H4JPTYHKE44S4Z1FY","type":"SOURCE","tag":"app","isSatisfied":true}`

They are missing 
- hash type
- host name
- layer type
- annotation type
- signature
- timestamp